core/sqf/sql/scripts/install_local_hadoop

```diff
@@ -1534,7 +1534,7 @@ fi
 
 cd $MY_SW_ROOT
 
-if [ ! -r $YARN_HOME/lib/native/libhdfs.so ]; then
+if [ -d $MY_LOCAL_SW_DIST && ! -r $YARN_HOME/lib/native/libhdfs.so ]; then
   echo "------------------------------------------------------------"
   echo "-- WARNING: libhdfs.so is not present on this system. Please"
   echo "--          build it, otherwise Trafodion will not compile."
```

Comment:
The libhdfs library is needed at both build and run time.  The way we have things setup, we should be creating this directory at build time, and copying the file at run time.  There may be a situation where somehow the library goes missing.  To avoid confusion to people who are trying out the product and should follow the build instructions from those who actually maintain this script, I will leave the message in but only display it if VERBOSE is enabled.  During debugging, if someone is having issue, then this information maybe useful.